src/dotnet/commands/dotnet-publish/Program.cs

```diff
@@ -43,6 +43,11 @@ public static int Run(string[] args)
             CommandOption versionSuffixOption = app.Option(
                $"--version-suffix <{LocalizableStrings.VersionSuffixOption}>", LocalizableStrings.VersionSuffixOptionDescription,
                 CommandOptionType.SingleValue);
+
+            CommandOption profileProjOption = app.Option(
+               $"--profile <{LocalizableStrings.ProfileProjOption}>", LocalizableStrings.ProfileProjOptionDescription,
+                CommandOptionType.SingleValue);
```

Comment:
You can see how to do multiple values here: https://github.com/dotnet/cli/blob/rel/1.0.0/src/dotnet/commands/dotnet-restore/Program.cs#L33.